File: DoNet6.DriverStandings/DotNet6.DriverStandings.Api/ViewModels/Race.cs
using DotNet6.DriverStandings.Domain.Model;
using System.Reflection;
using static DotNet6.DriverStandings.Api.ViewModels._Base;

namespace DotNet6.DriverStandings.Api.ViewModels
{
    public class RaceRequest : _BaseRequest
    {
        public int RaceId { get; set; }
    }
    public class RaceResponse : _BaseResponse
    {
    }

    public class RaceResponseList : _BaseListResponse<Race>
    {
    }
}

File: Angular.DriverStandings/DriverStandings-App/src/app/_services/_transferObjects/Race.ts
import { Race } from "src/app/_models/Race";
import { _BaseListResponse, _BaseRequest, _BaseResponse } from "./_Base";

export class RaceRequest extends _BaseRequest {
  public RaceID: number = 0;
}

export class RaceResponse extends _BaseResponse {
}

export class RaceResponseList extends _BaseListResponse<Race> {
}
